docs/user-documentation/extending.md

```diff
@@ -0,0 +1,13 @@
+# Extending Islandora
+ 
+Because Islandora 8 is very tightly integrated with Drupal, many use cases that required custom Islandora modules in previous versions can now be solved with a combination of configuration and modules from the Drupal community. Other use cases may still require custom code, but are not common enough, or complete enough, for that custom code to be a part of the core software. 
+ 
+Both scenarios are covered by the [Islandora Cookbook](https://github.com/Islandora-Labs/Islandora-Cookbook), which contains an annotated list of custom Islandora tools and modules developed by our community, a list of Druapl contributed modules that may be useful in an islandora context, and a set of _recipes_ which each provide a detailed outline for solving a particular use case. 
```

Comment:
Spelling: Drupal

" ...a list of Druapl contributed modules ..."

Reply:
oops! Good catch, @librarychik. Any chance you could make a pull request to fix that? I have so many branches open spread across different pages that I can't "rebase" (delete my repo and refork 😉 ), and I can't seem to get a new commit against this one without dragging a bunch of cruft.

Reply:
Happy to help.  I can't see a way to do a PR, can you point me in the right direction?

Reply:
@librarychik to do it in the GUI:

1. Click "Fork" (up at the top of the page here) to make your own fork. 
2. Find the page in your fork (/documentation/blob/master/docs/user-documentation/extending.md) and edit in the browser.
3. Commit the changes to your fork.
4. Go back to the home page of that repo in your fork and click "new pull request." The template should make it clear from there.

Reply:
Okay, done.  (hopefully correctly 😄 )

Reply:
Awesome! https://github.com/Islandora/documentation/pull/1357 looks good to me. We'll need a Committer to swoop in and hit the merge button. @whikloj ?

Reply:
I have you covered.